src/Tools/Common/ProcessNativeMethods/ProcessNativeMethods.cs

```diff
@@ -0,0 +1,126 @@
+// using Microsoft.Diagnostics.NETCore.Client;
+// using Microsoft.Tools.Common;
+using System;
+// using System.CommandLine;
+// using System.CommandLine.Invocation;
+// using System.CommandLine.IO;
+// using System.Collections.Generic;
+// using System.Diagnostics;
+// using System.Linq;
+using System.Runtime.InteropServices;
+// using System.Text;
+// using Process = System.Diagnostics.Process;
+// using System.IO;
+// using System.ComponentModel;
+// using System.Threading.Tasks;
+// using Microsoft.Diagnostics.Tools.Trace.CommandLine;
+// using System.CommandLine.Binding;
+
+namespace Microsoft.Internal.Common.Utils
```

Comment:
This was code was also copied from projectkudu right? We should add a comment saying so.